src/main/java/car/Car.java

```diff
@@ -30,4 +55,9 @@ public int getLocation(){
     private int getRandomNumber(){
         return randomNumber.getRandomNumber();
     }
+
+
+    public int maxLocation(int location) {
+        return this.location > location ? this.location : location;
```

Comment:
```suggestion
        Math.max(this.location, location);
```

추가로 우승자를 구하는 핵심로직중 하나이므로 테스트코드를 작성해보시면
좋을것 같습니다.